### ISSUE TITLE
User lost after sending signing request

### ISSUE THREAD

**Issue by bigtiger** [docusealco/docuseal#55]
We're using Docuseal in a self-hosted (Docker) configuration.

Last week I created a team and invited **Jack**. Today Jack logged in uploaded a document and marked it for signatures.

Despite having SMTP configured via [PostmarkApp](postmarkapp.com/) an email was not sent. When we went to investigate Jack wondered aloud whether the email setting I configured were only set up for me? When we went to verify that Jack tried to login and it took him through the account sign-up workflow. Despite having created and account last week, logging in this morning and creating a submission.

When I as the Team Owner log-in I still see Jack as part of the team. Jack finished the account sign-up and now appears to have an account divorced from my team.

Happy to provide more details and try to work through this but it was hit to our confidence.

P.s. When I signed in and saw the submission I was able to create old Jack's form and received an email stating it was complete.

**Comment by AlexandrToorchyn** [comment]
Hi @bigtiger, from what you've described above it seems like your docuseal database has been wiped out so when Jack tried to sign in again he was requested to go through the sign up process (creating a new account).

May I ask you which docker run command are you using? Are you using sqlite database or your own hosted postgresql to run docuseal?

I'd be happy to help to investigate the issue but so far it seems like some docker misconfiguration which led to a completely new database being created.

Regarding the SMTP - can you please run `docker logs -f docuseal_container_name` next time you send an email - those logs might help to debug the SMTP issue (use `docker ps` to find your docuseal container name).

**Comment by bigtiger** [comment]
Well, this is embarrassing. 😅

I dug around on the console and everything looked great. I went back to the user just now and we determined they had autocompleted `docuseal` from their browser and wound up at your hosted site rather than our self-hosted version.

Hence why they needed to create an account and couldn't find our templates.

The SMTP issue may still be open but I will wait until I have more details and open a separate ticket.

Thanks @AlexandrToorchyn!
